docusaurus/docs/Android/03-ui/04-components/03-message-list.md

```diff
@@ -24,62 +77,113 @@
 | --- | --- |
 |![Message_options_in light mode](../../assets/message_options_light.png)|![Message options in dark mode](../../assets/message_options_dark.png)|
 
-Most of the actions work out of the box but you can change their behavior using different listeners and handlers:
+If you're not going to use out of the box `MessageListViewModel` with it's handlers or just want to override action handlers you should define these handlers:
```

Comment:
```suggestion
Default action handlers are set after binding ViewModel with the View. You can customize default behavior by overriding each of the following handlers:
```

Reply:
and maybe before that - what do you think about explaining what handlers are?

Reply:
I think in the code sample there are nice comments which explain